Adyen/Analytics/AnalyticsProvider/TelemetryData.swift

```diff
@@ -0,0 +1,72 @@
+//
+// Copyright (c) 2022 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Foundation
+
+internal struct TelemetryData: Encodable {
+
+    // MARK: - Properties
+
+    internal var version: String? {
+        Bundle(for: AnalyticsProvider.self).infoDictionary?["CFBundleShortVersionString"] as? String
+    }
+
+    internal let channel: String = "iOS"
+
+    internal var locale: String {
+        let languageCode = Locale.current.languageCode ?? ""
+        let regionCode = Locale.current.regionCode ?? ""
+        return "\(languageCode)_\(regionCode)"
+    }
+
+    internal let userAgent: String? = nil
+
+    internal var deviceBrand: String {
+        var systemInfo = utsname()
+        uname(&systemInfo)
+        let machineMirror = Mirror(reflecting: systemInfo.machine)
+        return machineMirror.children.reduce("") { identifier, element in
+            guard let value = element.value as? Int8, value != 0 else { return identifier }
+            return identifier + String(UnicodeScalar(UInt8(value)))
+        }
+    }
+
+    internal var systemVersion: String {
+        UIDevice.current.systemVersion
+    }
+
+    internal var referrer: String {
+        Bundle.main.bundleIdentifier ?? ""
+    }
+
+    internal var screenWidth: Int {
+        Int(UIScreen.main.bounds.width)
+    }
```

Comment:
should you be using `UIScreen.main.nativeBounds` instead of `UIScreen.main.bounds`?:
`UIScreen.main.bounds` change with device orientation, but `UIScreen.main.nativeBounds` is the physical device width regardless of orientation.

Reply:
Hmm this is interesting. Let me check with the data team to see what we actually need. It may be interesting for us to know if there's any merchant using our product in a landscape orientation. Thanks!

Reply:
Change to use nativeBounds. We're gonna include an `orientation` property later.